readthedocs/projects/tasks.py

```diff
@@ -1396,10 +1396,34 @@ def warn(self, msg):
                     f'domain->name',
                 )
                 continue
+
+            # HACK: This is done because the difference between
+            # ``sphinx.builders.html.StandaloneHTMLBuilder``
+            # and ``sphinx.builders.dirhtml.DirectoryHTMLBuilder``.
+            # They both have different ways of generating HTML Files,
+            # and therefore the doc_name generated is different.
+            # More info on: http://bit.ly/sphinx-builders-info
+            # Also see issue: https://git.io/fjyft
```

Comment:
The original URLs should be just fine. And for the issue, you can just put the number.

Reply:
Updated the PR.